Xamarin.Forms.Platform.iOS/ContextScrollViewDelegate.cs

```diff
@@ -240,7 +240,7 @@ void ClearCloserRecognizer(UIScrollView scrollView)
 				return;
 
 			var cell = GetContextCell(scrollView);
-			cell.ContentCell.RemoveGestureRecognizer(_closer);
+			cell?.ContentCell?.RemoveGestureRecognizer(_closer);
```

Comment:
This looks like it avoids the exception. A variation I'm curious about would be to pass the ContentCell into this method from where it's created inside *WillEndDragging* so something like this

This way we make sure to remove the Gesture Recognizer from the contentcell

```C#
while (view.Superview != null)
					{
						view = view.Superview;
						var table = view as UITableView;
						if (table != null)
						{
							UITableViewCell contentCell = null;
							NSAction close = () =>
							{
								if (UIDevice.CurrentDevice.CheckSystemVersion(8, 0))
									RestoreHighlight(scrollView);

								IsOpen = false;
								scrollView.SetContentOffset(new PointF(0, 0), true);

								// pass the contentCell in since we have a reference to it here								
								ClearCloserRecognizer(scrollView, contentCell);
								contentCell = null;
							};

							_table = table;
							_globalCloser = new GlobalCloseContextGestureRecognizer(scrollView, close);
							_globalCloser.ShouldRecognizeSimultaneously = (recognizer, r) => r == _table.PanGestureRecognizer;
							table.AddGestureRecognizer(_globalCloser);

							_closer = new UITapGestureRecognizer(close);
							var cell = GetContextCell(scrollView);
							cell.ContentCell.AddGestureRecognizer(_closer);
							contentCell = cell.ContentCell;
						}
					}
```